src/pages/Swap/AdvancedSwapMode/AdvancedSwapMode.component.tsx

```diff
@@ -80,7 +80,7 @@ export const AdvancedSwapMode = ({ children }: PropsWithChildren) => {
           />
         </AdvancedModeHeader>
         <TransactionsWrapper
-          maxHeight={`${tradesWrapper?.current?.clientHeight ? tradesWrapper?.current?.clientHeight - 100 : 560}px`}
+          maxHeight={`${tradesWrapper?.current?.clientHeight ? tradesWrapper?.current?.clientHeight - 85 : 570}px`}
```

Comment:
can we extract this and give it a name?

Reply:
fixed [e9e2e80](https://github.com/levelkdev/swapr-dapp/pull/1517/commits/e9e2e80f994c92702af7a2ba1071d5639d0d2236)